src/inmanta/ast/statements/define.py

```diff
@@ -403,6 +403,7 @@ def evaluate(self):
 
         if self.left[1] is not None:
             left_end = RelationAttribute(right, left, self.left[1])
+            left_end.target_annotations = self.annotations
```

Comment:
why distinguish target_annotation and source_annotation if a RelationAttribute can only have one?
(or rather, if we distinguish target_annotation from source_annotation, do we need syntax to define both on on either side?)

Reply:
because for some visualizations targets (right hand side) is required and for some others the source (left hand side) is required. source and target are also the terms tosca uses

for example contained_in, depends_on, ...

Reply:
so how can one define a relation with both a left hand and right hand side? 
(e.g. a is_contained b, b monitors a)